src/controllers/interactivityController.js

```
@@ -204,13 +204,43 @@ async function showUserDetails({ data: username, report, respond }) {
     if (stats.commits > 0 || stats.pullRequests > 0 || stats.issues > 0) {
       text += `• *${repo}*\n`;
       text += `  Commits: ${stats.commits} | PRs: ${stats.pullRequests} | Issues: ${stats.issues}\n`;
+      if (stats.linesAdded || stats.linesModified) {
+        text += `  Lines Added: ${stats.linesAdded || 0} | Lines Modified: ${stats.linesModified || 0}\n`;
+      }
+    }
+  }
+  
+  // Add AI analysis if available
+  if (report.aiAnalysis && report.aiAnalysis.contributors && report.aiAnalysis.contributors[username]) {
+    const analysis = report.aiAnalysis.contributors[username];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)